src/inputValidator/headerValidator/formatFunctions.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-console */
+/* eslint-disable import/prefer-default-export */
+import checkQuotes from '../../utils/checkQuotes.js';
+import { cmcdHeader, keyTypes } from '../..//utils/constants.js';
+import { createError } from '../../utils/error.js';
+
+export const checkKeyInCorrectHeader = (header, key, errors) => {
+  if (!cmcdHeader[header].includes(key)) {
+    console.log(`The key ${key} does not go under the header ${header}`);
+    errors.push(createError('incorrect-format', key));
+  }
+};
+
+export const isStringCorrect = (key, value, errors) => {
+  if ((keyTypes[key] === 'string' && !checkQuotes(value)) || (keyTypes[key] === 'token' && checkQuotes(value))) {
```

Comment:
Try to use cmcdTypes.string and cmcdTypes.token instead of "string" or "token" respectively.